docs/architecture/microservices/secure-net-microservices-web-applications/index.md

```diff
@@ -70,56 +121,21 @@ Popular external authentication providers and their associated NuGet packages ar
 | **Facebook**  | **Microsoft.AspNetCore.Authentication.Facebook**         |
 | **Twitter**   | **Microsoft.AspNetCore.Authentication.Twitter**          |
 
-In all cases, the middleware is registered with a call to a registration method similar to `app.Use{ExternalProvider}Authentication` in `Startup.Configure`. These registration methods take an options object that contains an application ID and secret information (a password, for instance), as needed by the provider. External authentication providers require the application to be registered (as explained in [ASP.NET Core documentation](/aspnet/core/security/authentication/social/)) so that they can inform the user what application is requesting access to their identity.
+In all cases, You have to complete an application registration procedure that's vendor dependent, and usually involves:
```

Comment:
```suggestion
In all cases, you must complete an application registration procedure that is vendor dependent and that usually involves:
```

Reply:
Why is it wrong to use the contraction here?